cypress/e2e/bookStore.cy.js

```diff
@@ -2,10 +2,22 @@
 
 describe('Book Store app', () => {
   before(() => {
-    
+    cy.visit('/login');
+    cy.get('#userName').type('haha.test11');
+    cy.get('#password').type('Passwprd!123');
+    cy.contains('.text-right button', 'Login').click();
+    cy.get('#userName-value').should('contain.text', '');
+    cy.assertPageUrl('/profile');
   });
 
-  it('', () => {
-    
+  it('Add Book', () => {
```

Comment:
the name should say what it does continuing the describe title. in your case it will be read as 'Book Store app' + 'Add Book'.
fix it in a way to describe the purpose of the test